src/rcx/src/ext.cpp

```diff
@@ -51,17 +51,10 @@ extern "C" {
 extern int Rcx_Init(Tcl_Interp* interp);
 }
 
-Ext::Ext() : odb::ZInterface()
-{
-  _ext = new extMain(5);
-  _tree = NULL;
-  logger_ = nullptr;
-}
-
-Ext::~Ext()
-{
-  delete _ext;
-}
+Ext::Ext()
+  : odb::ZInterface(),
+    _ext(std::make_unique<extMain>(5))
```

Comment:
It looks like extMain's constructor accepts a param called menuId that's completely unused. I think it would be useful to remove it in the PR, and potentially remove a magic 5 from this file and others.

Other than that LGTM! Thanks for the contribution

Reply:
Done! I didn't see in the developer docs, are squashes preferred or is it ok to stack commits that address review comments?

Reply:
Sounds like squashes are not necessary, resolving thread.